main.go

```diff
@@ -232,6 +233,10 @@ The commands work as following:
    >                                                                  If you provide '-' as log output, it prints resuts to stdout.
    >                                                                  To be able to filter for tests to run or skip, use one argument per test selector. Example: runtest --test-to-run=(TestTarget.)TestClass/testMethod --test-to-run=(TestTarget.)TestClass/testMethod (the value for 'TestTarget' is optional)
    >                                                                  The method name can also be omitted and in this case all tests of the specified class are run
+   ios runxctest [--xctestrun-file-path=<xctestrunFilePath>]  [--log-output=<file>] [--test-to-run=<tests>]... [--test-to-skip=<tests>]... [--env=<e>]... [options]                    Run a XCTest. The --xctestrun-file-path specifies the path to the .xctestrun file to configure the test execution.
```

Comment:
We can remove the `[--test-to-run=<tests>]... [--test-to-skip=<tests>]... [--env=<e>]... [options]` args now.